utils/rpms/daos.spec

```diff
@@ -129,7 +130,12 @@ Requires: %{name} = %{version}-%{release}
 Requires: %{name}-client = %{version}-%{release}
 Requires: spdk-tools
 Requires: ndctl
-Requires: ipmctl
+# needed to set PMem configuration goals in BIOS through control-plane
+%if 0%{?is_opensuse}
+Requires: ipmctl < 02.00.00.3809
```

Comment:
Shouldn't we (instead? or at least in addition to this change) open a task to push upstream to update or build our own package with a newer/latest/el7-matching version rather than force a lower version?

Reply:
@brianjmurrell: We should land this ASAP, as it prevents a serious issue. If a user runs one of the bad versions of ipmctl on a system with Apache Pass DCPMMs, the modules will be put into a state that requires them to be physically pulled in order to restore them. I asked @tanabarr to proceed with this approach for the moment, with the understanding that we'll update the opensuse requirement if/when upstream gets around to packaging a newer version. In the interim, it will protect users on both platforms and the only downside is that users on leap15 won't be able to use Barlow Pass modules.